src/metrics.go

```diff
@@ -28,6 +28,7 @@ var metricsDefinition = map[string][]interface{}{
 	"system.usedMemoryPeakBytes":    {"used_memory_peak", metric.GAUGE},
 	"system.usedMemoryLuaBytes":     {"used_memory_lua", metric.GAUGE},
 	"system.totalSystemMemoryBytes": {"total_system_memory", metric.GAUGE},
+	"system.maxmemory":              {"maxmemory", metric.GAUGE},
```

Comment:
Is this metrics unit Bytes? The unit should be added at the end as per the other memory metrics

Reply:
> maxmemory: The value of the maxmemory configuration directive
> https://redis.io/commands/info

It's not explicitly stated in the official reference - I'm guessing it's bytes, just like total_system_memory, etc.

Tried:
```
config get maxmemory
maxmemory
0
...
config set maxmemory 128M
+OK

config get maxmemory '
maxmemory
128000000

info
...
used_memory_lua_human:37.00K
maxmemory:128000000
maxmemory_human:122.07M
maxmemory_policy:noeviction
...
```

Reply:
I would also put the Bytes suffix like the other metrics